src/main/java/org/altbeacon/beacon/service/RunningAverageRssiFilter.java

```diff
@@ -17,7 +17,7 @@
 
     private static final String TAG = "RunningAverageRssiFilter";
     public static final long DEFAULT_SAMPLE_EXPIRATION_MILLISECONDS = 20000; /* 20 seconds */
-    private static long sampleExpirationMilliseconds = DEFAULT_SAMPLE_EXPIRATION_MILLISECONDS;
+    protected static long sampleExpirationMilliseconds = DEFAULT_SAMPLE_EXPIRATION_MILLISECONDS;
```

Comment:
It looks like this may only be needed to have higher access for testing. Instead of exposing this as `protected`, which makes it exposed in public documentation, could we keep it `private` and use a getter with package-private access instead?

```java
    @RestrictTo(TESTS)
    static long getSampleExpirationMilliseconds() {
        return sampleExpirationMilliseconds;
    }
```